src/content/merge-queue/pause.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: Pausing the Merge Queue
```

Comment:
```suggestion
title: Pausing the Merge Queues
```

Reply:
I'll keep it single so it relies to the "product".